cluster/charts/rook-ceph/templates/deployment.yaml

```diff
@@ -94,11 +94,31 @@ spec:
         - name: ROOK_DISABLE_DEVICE_HOTPLUG
           value: "{{ .Values.disableDeviceHotplug }}"
         - name: ROOK_CSI_ENABLE_RBD
-          value: "{{ .Values.enableCsiRbdDriver }}"
+          value: "{{ .Values.csi.enableRbdDriver }}"
```

Comment:
You might need this check around the entire csi section in case they don't have that in their values:
```
{{- if .Values.csi }}
```

Reply:
done, forgot to add this check